presto-native-execution/src/test/java/com/facebook/presto/spark/TestPrestoSparkNativeSimpleQueries.java

```diff
@@ -125,4 +126,38 @@ public void testJsonFileBasedFunction()
         assertQuery("SELECT json.test_schema.map(json.test_schema.array_constructor(linenumber), json.test_schema.array_constructor(linenumber)) FROM lineitem", "SELECT map(array_constructor(linenumber), array_constructor(linenumber)) FROM lineitem");
         assertQuery("SELECT json.test_schema.map_entries(json.test_schema.map(json.test_schema.array_constructor(linenumber), json.test_schema.array_constructor(linenumber))) FROM lineitem", "SELECT map_entries(map(array_constructor(linenumber), array_constructor(linenumber))) FROM lineitem");
     }
+
+    @Test
+    public void testAggregationCompanionFunction()
+    {
```

Comment:
The updated test seems to be representative of the feature's application. 

nit: If companion function is a *niche* feature, I recommend adding a doc-str (comment) on what the test is doing; otherwise, ignore this comment. :)